### PR TITLE
Added youtube link for web development #1128

### LINKED ISSUE #1128: [Add] <I would like to add a youtube channel to learn web development>
### Site URL

https://www.youtube.com/@ApoorvGoyalMain

### Description of the above provided site

This youtube will help us to learn web development in a most interesting and understandable way.

### Category

Youtube

### Sub Category

Youtube

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by iqrafirdose**
Added a youtube link for web development.

## Fixes Issue

Closes #1128

## Changes proposed

Added a youtube channel for learning web development.

### CODE REVIEW

**Review by CBID2 — DISMISSED** [review]
LGTM! :)

File: database/youtube/web-development.json [hunk 1]
```diff
@@ -191,4 +191,13 @@
     "subcategory": "web-development",
     "language": "english"
   }
```
Comment (rupali-codes) [inline database/youtube/web-development.json:193]:
```suggestion
  },
```

File: database/youtube/web-development.json [hunk 2]
```diff
@@ -191,4 +191,13 @@
...
+  {
+    "name": "Apoorv Goyal",
+    "description": "Aprrov Goyal channel has web development boot camps, Git and Github playlist, Linux playlist, and Learn by doing series playlist. If you visit this channel once, you will go out of this with lots of learnings.",
+    "url": "https://www.youtube.com/@ApoorvGoyalMain",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
+  }
 ]
+
```
Comment (rupali-codes) [inline database/youtube/web-development.json:203]:
```suggestion
```

File: database/youtube/web-development.json [hunk 3]
```diff
@@ -190,5 +190,14 @@
...
-  }
+  },
+  {
+    "name": "Apoorv Goyal",
+    "description": "Aprrov Goyal channel has web development boot camps, Git and Github playlist, Linux playlist, and Learn by doing series playlist. If you visit this channel once, you will go out of this with lots of learnings.",
+    "url": "https://www.youtube.com/@ApoorvGoyalMain",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
+  },
```
Comment (rupali-codes) [inline database/youtube/web-development.json:201]:
```suggestion
  }
```

File: database/youtube/web-development.json [hunk 4]
```diff
@@ -190,5 +190,14 @@
...
+  {
+    "name": "Apoorv Goyal",
+    "description": "Aprrov Goyal channel has web development boot camps, Git and Github playlist, Linux playlist, and Learn by doing series playlist. If you visit this channel once, you will go out of this with lots of learnings.",
+    "url": "https://www.youtube.com/@ApoorvGoyalMain",
+    "category": "youtube",
+    "subcategory": "web-development",
+    "language": "english"
   }
 ]
++
```
Comment (rupali-codes) [inline database/youtube/web-development.json:203]:
```suggestion
```

**Comment by rupali-codes** [comment]
@iqrafirdose please resolve merge conflicts

**Review by rupali-codes — COMMENTED** [review]
You have resolved conflicts wrongly...please [read this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

**Comment by iqrafirdose** [comment]
> You have resolved conflicts wrongly...please [read this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)

Sorry, I am a beginner in open-source. It is not showing to resolve conflicts now.  
linkshub1

**Comment by rupali-codes** [comment]
> > You have resolved conflicts wrongly...please [read this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)
> 
> Sorry, I am a beginner in open-source. It is not showing to resolve conflicts now. linkshub1

its fine, lemme help you out, but please be careful from next time

File: database/youtube/web-development.json [hunk 5]
```diff
@@ -192,6 +192,11 @@
     "language": "english"
   },
   {
+
+    "name": "Apoorv Goyal",
+    "description": "Aprrov Goyal channel has web development boot camps, Git and Github playlist, Linux playlist, and Learn by doing series playlist. If you visit this channel once, you will go out of this with lots of learnings.",
+    "url": "https://www.youtube.com/@ApoorvGoyalMain",
+=======
```
Comment (rupali-codes) [inline database/youtube/web-development.json:199]:
```suggestion
    "category": "youtube",
    "subcategory": "web-development",
    "language": "english"
  },
  {
```

**Review by CBID2 — DISMISSED** [review]
LGTM! :)

File: database/youtube/web-development.json [hunk 6]
```diff
@@ -191,6 +191,15 @@
     "subcategory": "web-development",
     "language": "english"
   },
+  {
+
```
Comment (rupali-codes) [inline database/youtube/web-development.json:195]:
```suggestion
```

File: database/youtube/web-development.json [hunk 7]
```diff
@@ -243,8 +252,9 @@
     "name": "CodeWithSadee",
     "description": "This YouTube channel features informative and engaging Web development tutorials for all skill levels.",
     "url": "https://www.youtube.com/@codewithsadee/",
+
```
Comment (rupali-codes) [inline database/youtube/web-development.json:255]:
```suggestion
```

**Pull request merged**